src/shared.ts

```diff
@@ -54,6 +54,23 @@ const injectScript = (params: null | LoadParams): HTMLScriptElement => {
   return script;
 };
 
+const reloadScript = (params: null | LoadParams): HTMLScriptElement => {
+  const queryString =
```

Comment:
The query param logic should be shared with `injectScript` so it gets updated together.